public/mockServiceWorker.js

```diff
@@ -194,7 +194,10 @@ async function getResponse(event, client, requestId) {
 
   switch (clientMessage.type) {
     case 'MOCK_SUCCESS': {
-      return delayPromise(() => respondWithMock(clientMessage), clientMessage.payload.delay)
+      return delayPromise(
```

Comment:
This file is included in `.prettierignore` but my IDE keeps formatting it and I cannot make it stop. Let's just format this for my sake 🙏